frontend/src/component/Dashboard/RecentActivity.jsx

```diff
@@ -25,24 +27,43 @@ export const RecentActivity = props =>
       {props.recentActivityList.length > 0
         ? props.recentActivityList.map(content => {
           const contentType = props.contentTypeList.find(ct => ct.slug === content.type) || {hexcolor: '', faIcon: ''}
-          return (
+
+          const recentActivityItemIcon = (
+            <div className='recentactivity__list__item__icon' style={{color: contentType.hexcolor}}>
+              <i className={`fa fa-${contentType.faIcon}`} />
+            </div>
+          )
+
+          const recentActivityItemName = (
+            <div className='recentactivity__list__item__name'>
+              {content.label}
+              {content.type === 'file' && (
+                <Badge text={content.fileExtension} customClass='badgeBackgroundColor' />
+              )}
+            </div>
+          )
+
+          if (content.type === 'folder' && props.roleIdForLoggedUser < 4) return (
             <div
```

Comment:
It is intentional to not have a Link in this case ?

Reply:
I think i not understand correctly this part because we have no "folder" in "Recent activity", i'm wrong ?

Reply:
we do.
that's where the issue come from.
Folders should not be a clickable link for users that don't have enough rights to open the folder advanced app.